exec/java-exec/src/test/java/org/apache/drill/exec/compile/TestClassTransformation.java

```diff
@@ -79,11 +86,10 @@ public void testCompilationNoDebug() throws CompileException, ClassNotFoundExcep
     ClassSet classSet = new ClassSet(null, cg.getDefinition().getTemplateClassName(), cg.getMaterializedClassName());
     String sourceCode = cg.generateAndGet();
     sessionOptions.setLocalOption(ClassCompilerSelector.JAVA_COMPILER_OPTION, ClassCompilerSelector.CompilerPolicy.JDK.name());
-
     sessionOptions.setLocalOption(ClassCompilerSelector.JAVA_COMPILER_DEBUG_OPTION, false);
-    @SuppressWarnings("resource")
```

Comment:
Is this warning displayed now?

Reply:
No, it is not displayed, after removing this line.